charts/studio/templates/configmap-studio-backend.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: studio-backend
+data:
+{{- range $k, $v := .Values.studioBackend.envVars }}
+  {{ $v.name }}: "{{ $v.value }}"
+{{- end }}
```

You can replace range with toYaml:

```suggestion
{{- toYaml .Values.studioBackend.envVars  | nindent 4 }}
```